### PR TITLE
Fix change anon set, add tests

### PR DESCRIPTION
**Pull request by benthecarman**

### CODE REVIEW

File: client-server-test/src/test/scala/com/lnvortex/server/RemixTest.scala [hunk 1]
```diff
@@ -54,8 +54,16 @@ class RemixTest
...
+      val changeA = coinsA.filter(_.isChange)
+      assert(changeA.size == 1, s"${changeA.size} != 1")
+      assert(changeA.forall(_.anonSet == 1))
+
+      val changeB = coinsB.filter(_.isChange)
+      assert(changeB.size == 2, s"${changeB.size} != 2")
+      assert(changeB.forall(_.anonSet == 1))
+
+      assert(coinsA.count(_.anonSet >= 3) == 1)
+      assert(coinsB.count(_.anonSet >= 2) == 2)
```
Comment (benthecarman) [inline client-server-test/src/test/scala/com/lnvortex/server/RemixTest.scala:66]: Should use ==

File: client/src/main/scala/com/lnvortex/client/VortexClient.scala [hunk 2]
```diff
@@ -94,12 +94,14 @@ case class VortexClient[+T <: VortexWalletApi](vortexWallet: T)(implicit
...
+      val byOutpoint = utxoDbs.groupBy(_.outPoint)
       coins.map { coin =>
-        val utxoOpt = utxoDbs.find(_.outPoint == coin.outPoint)
+        val utxoOpt = byOutpoint.get(coin.outPoint).flatMap(_.headOption)
         utxoOpt match {
           case Some(utxo) =>
             coin.copy(anonSet = utxo.anonSet, isChange = utxo.isChange)
-          case None => coin
+          case None =>
+            coin
```
Comment (benthecarman) [inline client/src/main/scala/com/lnvortex/client/VortexClient.scala:104]: Ping

File: client/src/main/scala/com/lnvortex/client/VortexClient.scala [hunk 3]
```diff
@@ -94,12 +94,14 @@ case class VortexClient[+T <: VortexWalletApi](vortexWallet: T)(implicit
       coins <- vortexWallet.listCoins()
       utxoDbs <- utxoDAO.findByOutPoints(coins.map(_.outPoint))
```
Comment (benthecarman) [inline client/src/main/scala/com/lnvortex/client/VortexClient.scala:95]: Made can rewrite to return a Map

File: client/src/main/scala/com/lnvortex/client/VortexClient.scala [hunk 4]
```diff
@@ -652,7 +654,7 @@ case class VortexClient[+T <: VortexWalletApi](vortexWallet: T)(implicit
         for {
           _ <- vortexWallet.broadcastTransaction(signedTx)
           _ <- vortexWallet
-            .labelTransaction(signedTx.txId, s"LnVortex Mix Anon set: $anonSet")
+            .labelTransaction(signedTx.txId, s"Vortex Anon set: $anonSet")
```
Comment (RiccardoMasutti) [inline client/src/main/scala/com/lnvortex/client/VortexClient.scala:657]:
```suggestion
            .labelTransaction(signedTx.txId, s"Vortex Anonimity set: $anonSet")
```

**Review by RiccardoMasutti — CHANGES_REQUESTED** [review]

File: client/src/main/scala/com/lnvortex/client/VortexClient.scala (hunk 4 above)
Comment (benthecarman) [reply client/src/main/scala/com/lnvortex/client/VortexClient.scala]: Reverted this change to be done in your PR

**Pull request merged**